src/db/postgres/lrsql/postgres/sql/ddl.sql

```diff
@@ -317,3 +317,29 @@ ALTER TABLE xapi_statement ADD COLUMN stored TIMESTAMPTZ
 -- :command :execute
 -- :doc Backfill `xapi_statement.stored` with the values from the payload
 UPDATE xapi_statement SET stored = (payload->>'stored')::timestamptz WHERE stored IS NULL;
+
+/* Migration 2023-05-11-00 - Convert timestamps for consistency */
+
+-- :name query-state-document-last-modified-is-timestamptz
+-- :command :query
+-- :result :one
+-- :doc Query to see if `state_document.last_modified` is a timestamp.
+SELECT * FROM information_schema.columns WHERE table_name = 'state_document' AND column_name = 'last_modified' and data_type = 'timestamp with time zone';
```

Comment:
Probably should break this query up into separate lines. Also why is the last `and` lowercase?

Reply:
62ce52092f060df54af2fa6469fb8f358c21de15